papers/donath/donath-pylira.rst

```diff
@@ -0,0 +1,423 @@
+of the resulting image is also degraded by background or instrumental
+measurement noise and non-uniform exposure.
+For short wavelengths and associated low intensities
+of the signal, the imaging process consists in recording individual photons arriving from
+a source of interest, often called "events".
+This imaging process is typically for x-ray and gamma-ray telescopes, but is also found
+in magnetic resonance imaging or fluorescence microscopy.
+For each individual photon the incident direction and energy is
+measured. Based on this information the event can be histogramed
+into two dimensional data structures to form an actual image.
+
+Because of the low intensities associated recording of individual
+events, the measured signal follows Poisson statistics. This imposes
+a non-linear relationship between the measured signal and true
+underlying intensity as well as a coupling of the signal with the
+measurement noise. Any statistically correct post-processing
+or reconstruction method of thus requires are
+careful treatment of the Poisson nature of the measured image.
+
+To maximise the scientific use of the data is is often desired
+to correct the degradation introduced by the imaging process.
+Besides correction for non-uniform exposure and background
+noise this also includes the correction for the "blurring"
+introduced by the point spread function (PSF) of the
+instrument, often called "deconvolution". Depending on whether
+the PSF of the instrument is known, one distinguishes between
+the "blind deconvolution" and "non blind deconvolution" process.
+For astronomical observations the PSF can often either be
+simulated, given a model of the detector or inferred
+directly from the data by observing far distant objects,
+which appear as point source to the instrument.
+
+While in other branches of astronomy deconvolution methods are already part
+of the standard analysis, such as the CLEAN algorithm for radio data, this
+is not the case for x-ray and gamma-ray astronomy. As any deconvolution method
+aims to enhance small scale structures in an image it becomes increasingly
+hard to solve for the regime of low signal to noise ratio.
+State of the art de-blurring algorithms based on deep convolutional neural networks
+show exceptional results on conventional image data,
+but they cannot be straightforwardly applied to astronomical counts data,
+because of a lack of ground truth and training data in general.
+
+
+Deconvolution Methods
+---------------------
+
+Richardson-Lucy
++++++++++++++++
+One of the first methods for deconvolution of images with Poisson noise was
+proposed by :cite:`Richardson1972` and :cite:`Lucy1974`. This method, named
+after the original authors, is often known as the *Richardson & Lucy* (RL)
+method. The method takes the fundamental statistical properties of the image
+into account and describes the measurement process as a "forward fold" model.
```

Comment:
They called it "iterative method"

Reply:
Replaced by: 

>...describes the measurement process as a "forward folded" modeling problem. An estimate of the true image is then reconstructed using an iterative, likelihood based optimization method.

My point was that the model description involves forward folding, while the solution method is iterative. I hope it's clearer now...